app/src/main/java/com/example/vybe/MyVibesActivity.java

```diff
@@ -25,6 +35,7 @@
     private Button addVibeEventBtn;
     private Button myMapBtn;
     private Button socialBtn;
+    private FirebaseFirestore db = FirebaseFirestore.getInstance();
```

Comment:
Yes we can do that I just put a reference in for now. In fact it wasn't even used in the VibeEvent method I created because I just passed in the CollectionReference but again we can refactor it when things start moving